package.json

```diff
@@ -21,7 +28,7 @@
   "version": "1.0.0",
   "private": true,
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
+    "test": "jest",
```

Comment:
- [ ] [OPTIONAL] You can also consider adding a --coverage flag to your test script here, this could help you analyze your test coverage